ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -21,7 +21,8 @@ type RayClusterSpec struct {
 	// EnableInTreeAutoscaling indicates whether operator should create in tree autoscaling configs
 	EnableInTreeAutoscaling *bool `json:"enableInTreeAutoscaling,omitempty"`
 	// AutoscalerOptions specifies optional configuration for the Ray autoscaler.
-	AutoscalerOptions *AutoscalerOptions `json:"autoscalerOptions,omitempty"`
+	AutoscalerOptions  *AutoscalerOptions `json:"autoscalerOptions,omitempty"`
+	ServiceAnnotations map[string]string  `json:"serviceAnnotations,omitempty"`
```

Comment:
How about `HeadServiceAnnotations`? RayService will create several Kubernetes service resources (e.g. Dashboard, Head, Serve). `ServiceAnnotations` may cause users to feel confused.

Reply:
Good point!